### PR TITLE
Asp core middleware

### PR DESCRIPTION
**Pull request by bkwdesign**
This is my attempt to add parallel projects alongside your original solution to make this work in .NET Core 2.1

Feature a clone of your AspComet-vs2010 (titled AspCoreCometware) that uses Newtonsoft for JSON handling and features the fluent middleware constructs commonly found in .net core apps.

The sample .netcore 2.1 web app I've added seems to behave like your original, except I tweaked the comet URL path to look like it does for salseforce  (e.g. "comet/45.0")

### CODE REVIEW

**Comment by bkwdesign** [comment]
I have not attempted to run the NCover specs against the new AspCoreCometware library

File: src/AspComet/MessageConverter.cs [hunk 1]
```diff
@@ -26,7 +26,14 @@ public static Message[] FromJson(HttpRequestBase request)
 
         private static string GetJsonFromMessageFieldIn(HttpRequestBase request)
         {
-            return request.Form["message"];
+            try
+            {
+                return request.Form["message"];
```
Comment (nmosafi) [inline src/AspComet/MessageConverter.cs]: Strange the documentation says that `NameValueCollection` wouldn't throw if the key is missing, it should return null

Comment (bkwdesign) [reply src/AspComet/MessageConverter.cs]: i wasn't sure if the NCOver license allowed me to install a copy.. after looking at their outdated website I started to wonder what NCover might do to my V.S. 2017 installation.

So, I was hesitant to disturb your existing work. However, I'll be more than happy to do an in-place update to the project(s) and do a new pull request, if you prefer.

I also wasn't sure if people might prefer the older option in some scenarios, for whatever reason. Let me know what you prefer.

Comment (bkwdesign) [reply src/AspComet/MessageConverter.cs]: regading request.Form .. maybe the .NET Core version of the HttpRequest is more finicky?
I wish I had written down the exception I was getting,

**Comment by bkwdesign** [comment]
I wasn't sure if the NCover license allowed me to install a copy.. after looking at their outdated website I started to wonder what NCover might do to my V.S. 2017 installation.

So, I was hesitant to disturb your existing work. However, I'll be more than happy to do an in-place update to the project(s) and do a new pull request, if you prefer.

I also wasn't sure if people might prefer the older option in some scenarios, for whatever reason. Let me know what you prefer.

**Comment by nmosafi** [comment]
I don't use it any more myself, so I'm fine either way.  I'll merge this as is and if people complain about two versions we can take it from there

**Pull request merged**